web/src/lib/domain-logic/removeBusiness.ts

```diff
@@ -0,0 +1,24 @@
+import { UserData } from "@businessnjgovnavigator/shared/userData";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/explicit-function-return-type
+const removeKey = (key: string, { [key]: _, ...rest }) => rest;
+
+export const removeBusiness = ({
+  userData,
+  idToRemove,
+  newCurrentBusinessId,
+}: {
+  userData: UserData;
+  idToRemove: string;
+  newCurrentBusinessId: string;
+}): UserData => {
+  if (!Object.keys(userData.businesses).includes(newCurrentBusinessId)) {
+    throw "Business does not exist";
```

Comment:
[dust] Should there be an error object that gets created or a console log? What happens when you throw a string?